datafusion/physical-plan/src/aggregates/mod.rs

```diff
@@ -128,14 +128,14 @@ impl AggregateMode {
 #[derive(Clone, Debug, Default)]
 pub struct PhysicalGroupBy {
     /// Distinct (Physical Expr, Alias) in the grouping set
-    expr: Vec<(Arc<dyn PhysicalExpr>, String)>,
+    pub expr: Vec<(Arc<dyn PhysicalExpr>, String)>,
```

Comment:
A potentially more future proof API might be to make accessors for these fields

like 
```rust
pub fn null_expr(&self) -> &[...]
```

Reply:
ah, i just noticed that these properties do have accessors already so let me revert this chagne